lib/auth.ts

```diff
@@ -225,6 +225,11 @@ export function createLongLivedTokenAuth(
   hassUrl: string,
   access_token: string
 ) {
+  // Strip trailing slash.
+  if (hassUrl[hassUrl.length - 1] === "/") {
```

Comment:
Not sure what our target is for supporting browsers...
```suggestion
  if (hassUrl.endsWith("/")) {
```